src/item.h

```diff
@@ -636,8 +636,9 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
     /**
      * Whether the items is flammable. (Make sure to keep this in sync with
      * fire code in fields.cpp)
+     * @param threshold Item is flammable if it provides more fuel than threshold.
```

Comment:
More info need about `threshold`. Where is it read from? What are typical values etc